ex6.py

```diff
@@ -1,13 +1,19 @@
-x = f"There are {10} types of people."
+# left out assignment for types_of_people mentioned in intro
+types_of_people = 10
```

Comment:
This doesn't match up with the original Python 2 version and I believe the intent here is to indicate that you can add 'digit' strings (or whatever the Python 3.6 equivalent is). 

It's not as clear as the original so may need some tweaking but adding it as a variable 'types_of_people' matches the original intent.